src/modules/jcms_rest/tests/src/PathMimeTypeMapperTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Drupal\Tests\jcms_rest\Unit;
+
+use Drupal\Tests\UnitTestCase;
+use Drupal\jcms_rest\PathMimeTypeMapper;
+
+/**
+ * Class PathMimeTypeMapperTest
+ *
+ * @package Drupal\Tests\jcms_rest\Unit
+ */
+class PathMimeTypeMapperTest extends UnitTestCase {
+
+  /**
+   * @var \Drupal\jcms_rest\PathMimeTypeMapper
+   *   The class we're testing.
+   */
+  protected $pathMimeTypeMapper;
+
+  public function setUp() {
+    $this->pathMimeTypeMapper = new PathMimeTypeMapper();
+  }
+
+  public function dataProvider()
+  {
```

Comment:
I've just realised I need to add some more dynamic examples. Thanks, @giorgiosironi!